apps/dh/api-dh/source/DataHub.WebApi/Modules/MarketParticipant/Actor/Extensions/ActorDtoExtensions.cs

```diff
@@ -14,20 +14,14 @@
 
 using Energinet.DataHub.WebApi.Clients.MarketParticipant.v1;
 
-namespace Energinet.DataHub.WebApi.GraphQL.DataLoaders;
+namespace Energinet.DataHub.WebApi.Modules.MarketParticipant.Actor.Models;
 
-public class UserCacheDataLoader : CacheDataLoader<Guid, GetUserResponse>
+public static class ActorDtoExtensions
 {
-    private readonly IMarketParticipantClient_V1 _client;
-
-    public UserCacheDataLoader(
-        IMarketParticipantClient_V1 client,
-        DataLoaderOptions options)
-        : base(options) =>
-        _client = client;
-
-    protected override Task<GetUserResponse> LoadSingleAsync(
-        Guid key,
-        CancellationToken cancellationToken) =>
-        _client.UserAsync(key, cancellationToken);
+    public static string DisplayName(this ActorDto actorDto) => actorDto switch
+    {
+        null => string.Empty,
+        var actor when string.IsNullOrWhiteSpace(actor.MarketRole.EicFunction.ToString()) => actor.Name.Value,
+        var actor => $"{actor.MarketRole.EicFunction} • {actor.Name.Value}",
+    };
```

Comment:
question: is this actually used anywhere other than in `ActorType.cs`? if it is not and is not supposed to, then I would suggest to just move this logic to a computed field on `ActorType`.

Reply:
You are right, no need for the extension method